app/(tabs)/index.tsx

```diff
@@ -1,11 +1,74 @@
-import { View, Text } from "react-native";
+import { View, Text, StyleSheet, ScrollView } from "react-native";
+import ConfirmationCard from "../components/confirmationCard";
+import AppointmentCard from "../components/appointmentCard";
 
 const StartPage = () => {
     return (
-        <View>
-            <Text>Home Page</Text>
-        </View>
+        <ScrollView>
+            <View style={styles.greeting}>
+                <Text style={styles.greetingText}>Hallo Stephan!</Text>
+            </View>
+            <View style={{height: 40}}/>
+            <View style={styles.nextSteps}>
+                <Text style={styles.nextStepsHeadline}>Deine nächsten Schritte</Text>
+                <Text style={styles.nextStepsSub}>Alle Fortschritte anzeigen</Text>
+            </View>
+            <ScrollView style={styles.confirmationCardScrollView} horizontal showsHorizontalScrollIndicator={false}>
+                <ConfirmationCard/>
+                <View style={{width: 16}}/>
```

Comment:
Is this a hardcoded space to seperate the ``ConfirmationCard`` elements? If so, please remove it and simply use the ``gap: 16`` CSS attribute for ``confirmationCardScrollView``.